src/main/java/frc/robot/OI.java

```diff
@@ -0,0 +1,232 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInLayouts;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardLayout;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import java.util.Map;
+
+public class OI {
```

Comment:
Is there a good reason not to call this class `OperatorInterface` instead of `OI`?